### ISSUE TITLE
Marker names

### ISSUE THREAD

**Issue by paaljoachim** [WordImpress/google-maps-builder#88]
Marker names:

![screen shot 2015-08-23 at 12 46 44 am](https://cloud.githubusercontent.com/assets/5323259/9426181/756c32cc-4930-11e5-8638-6f21bdc1a751.png)

Marker: 1, 2, 3, 4 and 5.
Rename these to marker title instead.

**Comment by DevinWalker** [comment]
Unfortunately that functionality isn't built into CMB2 yet...

**Comment by DevinWalker** [comment]
I agree this is a good idea, but until CMB2 adds this functionality we're kinda left high and dry.

**Comment by paaljoachim** [comment]
Check the above link for feedback from Justin.

**Comment by DevinWalker** [comment]
Dang, I was hoping it would be received more kindly.
